### DISCUSSION TITLE
GoldenEye: Archives

### DISCUSSION THREAD

**Discussion by JCR64** [nzp-team/nzportable#889] [Map Releases]
A faithful remake of the Archives level from GoldenEye (N64). I've only tested it on FTE/PC and as it's quite large (and badly optimized) it might have problems on other platforms.

Easter Eggs:
- Original music can be activated in the room you start in (hint: posters)
- Recovering the helicopter black box enables a buyable ending and more music

Credits: 
- Rare

# [DOWNLOAD](https://archive.org/details/nzp_ge64_archives)

![screen1](https://github.com/user-attachments/assets/2f71d27a-aa8f-4569-a2a8-46e2cded4ac1)

![screen2](https://github.com/user-attachments/assets/0b70e6a6-82b0-41b4-9a77-8ae5302cd7c7)

# [Video Playthrough](https://www.youtube.com/watch?v=LGhN8HwBSEc)

**Comment by ToonLink0001** [comment]
this safe to download?

**Reply by JCR64** [reply]
I hope so! You can check the contents of the .zip file via archive.org [here](https://ia600602.us.archive.org/view_archive.php?archive=/33/items/nzp_ge64_archives/ge64_archives.zip). However like I said in the release description, I've only tested it on PC so I can't say for certain for 3DS/PSP.

**Comment by Evilslayer506** [comment]
Hello, i was wondering how you were able to get not only an easter egg song sorted, but also a simple easter egg as well, just curious

**Reply by JCR64** [reply]
There's documentation on [https://docs.nzp.gay](https://docs.nzp.gay/landing/#mapping-modding-documentation) that will help guide you but imo nothing beats loading one of the included maps of NZP into TrenchBroom to see how they operate e.g. [ndu.map](https://github.com/nzp-team/assets/blob/main/source/maps/ndu/ndu.map) and how it uses exploding barrels for its easter egg.

A basic overview of what is being done in GoldenEye Archives:

`trigger_interact` 
-target-> `ambient_generic` for a sound effect
-target->  `game_counter` to count the interactions
-target->  `game_songplay` to play a song once it reaches counter's limit
-killtarget-> `func_door`  to remove something if needed

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as this is not a direct download link.

**Comment by nonMat** [comment]
This was so surprisingly fun! 
It was on the easier side but I had never played NZ:P like that! 
<img width="1920" height="1080" alt="GoldenEye: Archives" src="https://github.com/user-attachments/assets/1c14287f-7638-4eef-b896-5f183f70519c" />
